onmt/modules/multi_headed_attn.py

```diff
@@ -189,4 +178,4 @@ def unshape(x):
                   query_len, key_len)[:, 0, :, :] \
             .contiguous()
 
-        return output, top_attn
+        return output, top_attn
```

Comment:
you got to fix flake but I think you're change is not good anyway, see my other comment.